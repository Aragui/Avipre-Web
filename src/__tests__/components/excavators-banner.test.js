import {render} from '@testing-library/react'

import ExcavatorsBanner from '../../components/excavators-banner'

describe('<ExcavatorsBanner />', () => { 
    const {container} = render(<ExcavatorsBanner />)

    test('Render Banner', () => { 
        expect(container).toMatchSnapshot();
     })
 })