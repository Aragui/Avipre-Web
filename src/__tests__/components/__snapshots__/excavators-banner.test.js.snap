// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ExcavatorsBanner /> Render Banner 1`] = `
<div>
  <div
    class="excavators-bannerstyles__BannerWrapper-sc-1qbd7sq-0 fbhPuU container-fluid"
    id="maquinaria"
  >
    <div
      class="row py-5"
    >
      <div
        class="col-sm-12 col-md-12 col-lg-6 d-flex justify-content-center align-items-center"
      >
        <img
          alt="Renta de retroexcavadoras"
          class="banner-cover"
          src="/images/excavadora.png"
        />
      </div>
      <div
        class="col-sm-12 col-md-12 col-lg-6 py-3 ps-3 d-flex flex-column justify-content-center align-items-center"
      >
        <div>
          <h2
            class="banner-title"
          >
            RENTA de Retroexcavadoras
          </h2>
          <p
            class="banner-subtitle mt-5"
          >
            Hacemos trabajos de:
          </p>
          <ul
            class="banner-list"
          >
            <li>
              Limpieza de terrenos y desmontes.
            </li>
            <li>
              Fosas.
            </li>
            <li>
              Cisternas.
            </li>
            <li>
              Piscinas.
            </li>
            <li>
              Zanjas.
            </li>
          </ul>
          <a
            class="btn btn-warning mt-5"
            href="tel:9991295507"
          >
            9991295507
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
